Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/partials/media-detail.xhtml

```diff
@@ -16,14 +16,45 @@
         xmlns:f="http://xmlns.jcp.org/jsf/core"
         xmlns:fn="http://xmlns.jcp.org/jsp/jstl/functions"
         xmlns:p="http://primefaces.org/ui"
-        xmlns:ui="http://xmlns.jcp.org/jsf/facelets">
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:a="http://xmlns.jcp.org/jsf/passthrough">
 
     <h:panelGroup id="mediaDetail" layout="block">
         <ui:param name="selectedGalleryMediaContent"
                   value="#{DataEditorForm.galleryPanel.getGalleryMediaContent(DataEditorForm.galleryPanel.lastSelection.key)}"/>
 
         <ui:fragment
                 rendered="#{mediaProvider.hasMediaViewVariant(selectedGalleryMediaContent) and (fn:startsWith(selectedGalleryMediaContent.mediaViewMimeType, 'video') or fn:startsWith(selectedGalleryMediaContent.mediaViewMimeType, 'audio'))}">
+
+            <p:outputPanel rendered="#{fn:startsWith(selectedGalleryMediaContent.mediaViewMimeType, 'audio') and DataEditorForm.galleryPanel.isAudioMediaViewWaveform()}">
+                <div id="audioWaveformTools" >
+                    <div>
+                        <label><h:outputText value="#{msgs.audioWaveformToolsZoom}"/>: <input type="range" min="0" max="250" value="0"/></label>
+                    </div>
+                    <div>
+                        <label><h:outputText value="#{msgs.audioWaveformToolsCenteredCursor}"/>: <input type="checkbox" checked="" value="autoCenter"/></label>
+                    </div>
+                    <div>
+                        <p:commandButton title="#{msgs.audioWaveformToolsJumpBackFiveSeconds}"
```

Comment:
These buttons should have the attribute `type="button"`, when they only trigger client side actions. Otherwise they will submit their form and cause an unnecessary update.

Reply:
Nice! It had worked for me as well without type by manually stopping the propagation. Now I've added the type and removed the stopping. Thanks for the suggestion.